cns/service/main.go

```diff
@@ -453,8 +453,13 @@ func main() {
 		return
 	}
 
+	nmaclient, err := nmagentclient.NewNMAgentClient("")
```

Comment:
what's the use of having empty param to this?

Reply:
In case we want to override nmagent client url.